jetson/power_logging/model/save_model_summary.py

```diff
@@ -0,0 +1,97 @@
+import torch
+import json
+from typing import Any 
+
+def load_model(model_name: str, model_repo: str) -> Any:
```

Comment:
Loading models is already defined in benchmark.py. I suggest using that implementation (as it definitely supports all the models we are interested in). I suggest moving it to some common module (e.g. model_utils.py), and importing it in both places.